modules/connectors/ninja_van/karrio/mappers/ninja_van/proxy.py

```diff
@@ -10,45 +10,75 @@ class Proxy(proxy.Proxy):
 
     def get_rates(self, request: lib.Serializable) -> lib.Deserializable[str]:
         response = lib.request(
-            url=f"{self.settings.server_url}/service",
+            url=f"{self.settings.server_url}/{self.settings.country_code}/1.0/public/price",
             data=lib.to_json(request.serialize()),
             trace=self.trace_as("json"),
             method="POST",
-            headers={},
+             headers={
+                "Accept": "application/json",
+                "Content-type": "application/json",
+                "Authorization": f"Bearer {self.settings.access_token}",
+            },
         )
 
         return lib.Deserializable(response, lib.to_dict)
-    
+
     def create_shipment(self, request: lib.Serializable) -> lib.Deserializable[str]:
         response = lib.request(
-            url=f"{self.settings.server_url}/service",
+            url=f"{self.settings.server_url}/{self.settings.country_code}/4.2/orders",
```

Comment:
change the country_code to generic property

Reply:
Done